appOPHD/UI/DetailMap.cpp

```diff
@@ -134,9 +131,12 @@ void DetailMap::draw() const
 			// Draw a beacon on an unoccupied tile with a mine
 			if (tile.mine() != nullptr && !tile.thing())
 			{
-				uint8_t glow = static_cast<uint8_t>(120 + std::sin(throbTimer.tick() / ThrobSpeed) * 57);
-				renderer.drawImage(mMineBeacon, position + NAS2D::Vector{0, -64});
-				renderer.drawSubImage(mMineBeacon, position + NAS2D::Vector{59, 15}, NAS2D::Rectangle<int>{{59, 79}, {10, 7}}, NAS2D::Color{glow, glow, glow});
+				constexpr NAS2D::Vector<int> offsetInTile{0, -64};
+				constexpr NAS2D::Vector<int> lightOffsetInTile{59, 15};
+				constexpr NAS2D::Rectangle<int> lightSubImageRect{{59, 79}, {10, 7}};
```

Comment:
Having the name `lightOffsetInTile` kind of raises the question what does `offsetInTile` refer to? I would assume it's `mineOffsetInTile`.